releases/v0.2.2-x86_64-linux.SHA256SUMS

```diff
@@ -0,0 +1,6 @@
+716840cfe02a270aff5bf1a02eda9c87946af231ad4ca5796a981ed0d9b1723f  v0.2.2-x86_64-linux/fedimint-cli
```

Comment:
I'm getting the same hashes

Reply:
![image](https://github.com/fedimint/fedimint/assets/1859166/b1d7d01f-8feb-49ba-8024-bb8a3f48abed)

Same as me :)